service/operator/framework.go

```diff
@@ -39,33 +40,45 @@ func newFramework(config Config) (*framework.Framework, error) {
 		}
 	}
 
-	var resourceGroupResource framework.Resource
+	var resourceGroupResource *resourcegroup.Resource
 	{
-		resourceGroupConfig := resourcegroup.DefaultConfig()
-		resourceGroupConfig.AzureConfig = config.AzureConfig
-		resourceGroupConfig.Logger = config.Logger
+		c := resourcegroup.DefaultConfig()
+		c.AzureConfig = config.AzureConfig
+		c.Logger = config.Logger
 
-		resourceGroupResource, err = resourcegroup.New(resourceGroupConfig)
+		resourceGroupResource, err = resourcegroup.New(c)
 		if err != nil {
 			return nil, microerror.Maskf(err, "resourcegroup.New")
 		}
 	}
 
-	var deploymentResource framework.Resource
+	var deploymentResource *deployment.Resource
 	{
-		deploymentConfig := deployment.DefaultConfig()
-		deploymentConfig.TemplateVersion = config.TemplateVersion
-		deploymentConfig.AzureConfig = config.AzureConfig
-		deploymentConfig.CertWatcher = certWatcher
-		deploymentConfig.CloudConfig = cloudConfig
-		deploymentConfig.Logger = config.Logger
-
-		deploymentResource, err = deployment.New(deploymentConfig)
+		c := deployment.DefaultConfig()
```

Comment:
I like the short `c` here, but I personally would appreciate new lines between logical blocks.

Reply:
This is 1:1 change. I only changed name. Where are the logical blocks?

Reply:
```
c := deployment.DefaultConfig()

<dependencies>

<settings>

<constructor magic>
```

Reply:
I tried to do that, but there are so many unrelated changes, I'd like to do that in separate PR.

Reply:
Easy. All good.